src/subscription_manager/utils.py

```diff
@@ -170,6 +170,12 @@ def get_terminal_width():
     """
     Attempt to determine the current terminal size.
     """
+    try:
+        import sys
+        if not sys.stdout.isatty():
+            return None
+    except Exception:
```

Comment:
afaik, the only case sys.stdout.isatty should fail is if sys.stdout if monkeypatch'ed (some unit tests likely do this), which means the test is wrong.

In other words, doesn't really need a try/except here.